src/main/java/uk/gov/laa/ccms/caab/bean/validators/client/AbstractClientAddressValidator.java

```diff
@@ -0,0 +1,27 @@
+package uk.gov.laa.ccms.caab.bean.validators.client;
+
+import org.springframework.validation.Errors;
+import uk.gov.laa.ccms.caab.bean.ClientDetails;
+import uk.gov.laa.ccms.caab.bean.validators.AbstractValidator;
+
+/**
+ * Abstract validator used for client address validation.
+ */
+public abstract class AbstractClientAddressValidator extends AbstractValidator {
+
+  protected void validateNoFixedAbode(ClientDetails clientDetails, Errors errors) {
+    if ((clientDetails.getCountry() != null && !clientDetails.getCountry().isEmpty())
```

Comment:
If you use the commons StringUtils.isEmpty(...) for all of these it would simplify the if statement a bit.
It checks for null and a length of 0.

Reply:
Amended using org.springframework.util.StringUtils StringUtils.hasText()